cg/meta/deliver.py

```diff
@@ -277,3 +280,28 @@ def create_delivery_dir_path(self, case_name: str = None, sample_name: str = Non
             delivery_path = delivery_path / sample_name
 
         return delivery_path
+
+    @staticmethod
+    def get_delivery_arguments(case_obj: models.Family) -> Set[str]:
+        """Translates the case data_delivery field to pipeline specific arguments."""
+        delivery_arguments: Set[str] = set()
+        requested_deliveries: List[str] = re.split("[-_]", case_obj.data_delivery)
+        if DataDelivery.FASTQ in requested_deliveries:
+            delivery_arguments.add(Pipeline.FASTQ)
+        if DataDelivery.ANALYSIS_FILES in requested_deliveries:
+            delivery_arguments.add(case_obj.data_analysis)
+        return delivery_arguments
+
+    @staticmethod
+    def get_delivery_scope(delivery_arguments: Set[str]) -> Tuple[bool, bool]:
+        """Returns the scope of the delivery, ie whether sample and/or case files were delivered."""
+        case_delivery = sample_delivery = False
+        for delivery in delivery_arguments:
+            if (
+                constants.PIPELINE_ANALYSIS_TAG_MAP[delivery]["sample_tags"]
+                and delivery in constants.ONLY_ONE_CASE_PER_TICKET
+            ):
+                sample_delivery = True
+            if constants.PIPELINE_ANALYSIS_TAG_MAP[delivery]["case_tags"]:
+                case_delivery = True
```

Comment:
Would this work:
```
case_delivery = True if constants.PIPELINE_ANALYSIS_TAG_MAP[delivery]["case_tags"]
```

Reply:
It would need an `else` which could overwrite the variable if it was set to true earlier in the loop. I think this is the most readable version of this function

Reply:
👍